kuma/users/jinja2/users/ban_user_and_cleanup_summary.html

```diff
@@ -41,7 +41,37 @@
       {% else %}
       <p>{{ _('The user did not have any revisions that were reported as spam.') }}</p>
       {% endif %}
-    </div>
+    </div><!-- /#revisions-reported-as-spam -->
+
+    <h5>{{ _('Deleted:')}}</h5>
+    <div id="revisions-deleted">
+      {% if actions_taken.revisions_deleted_list %}
+        <ul>
+          {% for revision in actions_taken.revisions_deleted_list %}
+            <li>
+              {{ revision.title }}
+            </li>
+          {% endfor %}
```

Comment:
I think it makes more sense to use the document title, and to link to it as well (with ``revision.document.get_absolute_url() ``), because admins have the option to revert an accidentally deleted page by following the link.